jax/_src/numpy/ufuncs.py

```diff
@@ -374,9 +374,41 @@ def exp(x: ArrayLike, /) -> Array:
   """
   return lax.exp(*promote_args_inexact('exp', x))
 
-@implements(np.log, module='numpy')
+
 @partial(jit, inline=True)
 def log(x: ArrayLike, /) -> Array:
+  """Calculate element-wise natural logarithm of the input.
+
+  JAX implementation of :obj:`numpy.log`.
```

Comment:
This sounds weird because `numpy.log` is also an implementation (of function `log`). How about:
```suggestion
  Calculates `log(1 + x).
```
?

Although, I see that other functions also use the same pattern: "JAX implementation of :obj:`numpy.xyz`."

Reply:
We use this pattern for every `jax.numpy` function, so I don’t think we should change it here. Each Jax function is the Jax implementation of the corresponding numpy function.